### DISCUSSION TITLE
Color of quotation marks in VSC

### DISCUSSION THREAD

**Discussion by mbart13** [arcticicestudio/nord#244] [Help & Support] [answered]
Hi All,

I recently switched from Webstorm to VSC and noticed my favorite theme looks different.
In Webstorm color of quotation marks is the same color as text, in VSC it's not, and looks rather strange.

Is there some custom setting I can change to make quotations in VSC look like in Webstorm?

Here is comparison: 

Webstorm
![obraz](https://user-images.githubusercontent.com/36601103/110612929-f3c54e80-8190-11eb-9c25-dcc1a7512be3.png)

VSC
![obraz](https://user-images.githubusercontent.com/36601103/110612970-ff187a00-8190-11eb-8ba0-fa9b1e860c8b.png)

**Comment by mbart13** [comment]
anyone?

**Comment by arcticicestudio** [comment]
Hi @mbart13 :wave:, thanks for your contribution :+1:

To be true, the rendering from VSC is the style that this theme aims for. It uses the correct colors for punctuation elements (quotes, equal sign) so that the actual string value can be better recognized.
Even when the JetBrains products are way more advanced in many aspects, the highlighting rules for HTML are a bit lean. There is currently no way to differ between those elements but only assign a color for “attribute values“, which also simply applies to the equal sign and quotes.

If you'd like to change the style for VSC, please read [the documentation about how to override styles with customizations][d].

[d]: https://www.nordtheme.com/docs/ports/visual-studio-code/customization

**Comment by mbart13** [comment]
I know this link, but how do I know which property I need to override?

**Comment by arcticicestudio** [comment]
There are no links added in Nord's documentation to the official VS Code documentation yet, but VS Code comes with tools like the [scope inspector][vsc-docs-si] that can be used to inspect [_TextMate_ syntax tokens and scopes][vsc-docs-tm_ts].

[vsc-docs-si]: https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#scope-inspector
[vsc-docs-tm_ts]: https://code.visualstudio.com/api/language-extensions/syntax-highlight-guide#textmate-tokens-and-scopes

**Comment by svengreb** [comment, marked as answer]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.

---

@mbart13 Thanks again for your contributions 🚀
Changes like this can be, like already mentioned, made by [customizing the theme to fit your needs](https://www.nordtheme.com/docs/ports/visual-studio-code/customization).
I‘ll close this discussion since this is a user-side related solution.
